.github/workflows/prune.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           organization: ${{ github.repository_owner }}
-          container: charts/${{ github.repository.name }}
```

Comment:
Reason: https://github.com/caas-team/sparrow/actions/runs/8119245352

I didn't want to open a dedicated PR for these two lines so let's do it here.

Reply:
:+1: